src/modules/auth/components/SetPasswordForm/SetPasswordForm.js

```diff
@@ -76,23 +77,18 @@ function SetPasswordForm({ onSubmit, recoveryPhrase }) {
             secureTextEntry
             feedback={errors.password?.message}
             status={errors.password ? 'error' : undefined}
-            defaultValues=""
-            value={password}
-            label={(
-              <span className="password-label-wrapper">
+            label={
+              <span className="password-label-wrappe ">
```

Comment:
@oskarleonard I believe this was meant to be `password-label-wrapper`

Reply:
Should be fixed now, great catch